### PR TITLE
Admins can launch tactical nukes at the station

### PR DESCRIPTION
**Pull request by Kmc2000**
### Intent of your Pull Request

https://www.youtube.com/watch?v=rN760628dm8

You can now launch nukes at the station from Centcom!

Refer to this video for information:

https://drive.google.com/file/d/1JJI5uRRw1wIB2dtHyx7WopzPfcZtRwmX/view?usp=sharing

It will rain REAL nuclear hellfire down on the station instead of playing a dumb meme cutscene. 

It comes in a few stages:
1:
Warning
2:
Blast
3:
Initial heatwave and pressure wave
4:
Ash falls
5: 
Fallout 

Points of note:
. Any unprotected mobs will be vaporized on the initial impact
. You can survive this! a hint on what to do is given in the announcement that accompanies the blast
. Fallout isn't like the weird, traditional radstorm. Instead it just deals rad damage over-time. You can still go out in it but expect to contract cancer.
. Sounds made by myself through various resources on YouTube
. This can only be triggered by admins due to its destructive nature.
. This does not replace the traditional nuke, as this simulates a nuke being detonated in the vicinity of ss13. Not onboard (as that'd just destroy it)
. Of course this isn't 100% realistic. It's cool though

To the naysayers:
It'll make our server more unique.

#### Changelog

:cl: Kmc2000
rscadd: Admins now have the ability to launch tactical nuclear weapons at all local Nanotrasen stations. No, I'm not an atomic playboy
rscadd: Admins! This verb is in the "Fun" tab, so you need that permission to use it.
/:cl:

### CODE REVIEW

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -326,7 +327,16 @@
 			feedback_add_details("admin_secrets_fun_used","QP")
 			log_admin("[key_name(usr)] made all SMESs powered", 1)
 			message_admins("<span class='adminnotice'>[key_name_admin(usr)] made all SMESs powered</span>")
-			power_restore_quick()
```
Comment (ThatLing) [inline code/modules/admin/secrets.dm]: nice

Comment (ThatLing) [inline code/modules/admin/secrets.dm]: nice

**Review by ThatLing — DISMISSED** [review]

**Review by ThatLing — APPROVED** [review]

**Comment by Jc1469** [comment]
I see an include for the weather type but not the actual weather .dm :thought_balloon:

**Comment by Notamaniac** [comment]
>using bandicam to record this abomination
>nuke in "airspace" sorry what we're in space there is no air
>no TACTICAL NUKE INCOMING cod soundbyte to even properly meme

**Comment by Kmc2000** [comment]
Shut up pseudo your opinion is invalidated until i receieve my fanart.

:)

**Comment by Nickvr628** [comment]
So what determines putting as much metal between yourself and the outside? Are places like the AI core safe or is it maint only for shelters?

**Comment by Kmc2000** [comment]
Spot on, any areas in maint with a space facing window or direct proximity to main halls are a no-go.

Maint itself is not safe from the shockwave, but some bits of it are.

**Comment by Time-Green** [comment]
You can hold a spacesuit in your hands and it'd protect you

**Comment by Kmc2000** [comment]
Logical...

Thought I removed that part :thinking:

**Pull request closed without merging**